images/tiler-db/config/update_postgresql.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# max_connections
+if [[ "${POSTGRES_DB_MAX_CONNECTIONS}X" != "X" ]]; then
```

Comment:
this if statement looks a bit weird to me? Why not just check if POSTGRES_DB_MAX_CONNECTIONS is set or so?

Reply:
It check if the env Var`POSTGRES_DB_MAX_CONNECTIONS` was set